examples/collision_detector/collision_detector.cc

```diff
@@ -42,7 +42,7 @@ class Agent
         }
     }
 
-    virtual void draw(sf::RenderTarget &target, sf::RenderStates states) const override
+    void draw(sf::RenderTarget &target, sf::RenderStates states) const override
```

Comment:
I know the ``override`` prefixes make this unnecessary but, personally I like having ``virtual`` at the start of virtual functions to make it extra-clear.

Reply:
So the point of this check is to mark functions as ``override`` if they need to be, but it does have the annoying side-effect of also removing the ``virtual`` prefixes when it does so... What do you think? I feel it is a useful thing to check for.

Reply:
Marking functions as ``override`` _is_ clearly valuable. I am happy to have my slightly-archaic internal style guide overriden on the virtual issue in exchange 😄

Reply:
Cool :-)